LoopUp.Siesta/RequestConfiguration/SiestaRequestOfT.cs

```diff
@@ -7,14 +7,14 @@ namespace LoopUp.Siesta.RequestConfiguration
     /// Base class for any request that expects data to be returned.
     /// </summary>
     /// <typeparam name="T">The expected data return type.</typeparam>
-    public abstract class SiestaRequestBase<T> : SiestaRequestBase
+    public abstract class SiestaRequest<T> : SiestaRequest
     {
         private readonly Type contentType;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="SiestaRequestBase{T}"/> class.
+        /// Initializes a new instance of the <see cref="SiestaRequest{T}"/> class.
         /// </summary>
-        protected SiestaRequestBase() => this.contentType = typeof(T);
+        protected SiestaRequest() => this.contentType = typeof(T);
     }
 }
 #pragma warning restore SA1649
```

Comment:
Unrelated to this PR, but you can remove this `#pragma` if you rename the file either `SiestaRequest{T}.cs` or ``SiestaRequest`T.cs``

Reply:
So I looked into naming conventions for generic class files, and this is seemingly what microsoft use. I am happy to go with one of the others you have recommended if you prefer?

Reply:
I would personally go with `SiestaRequest{T}.cs` - but I don't mind, either is better than removing a warning.

Reply:
This has been changed in a subsequent PR